src/server/scripts/Northrend/Naxxramas/boss_grobbulus.cpp

```diff
@@ -15,35 +15,51 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+/*
+ * Slime Spray timers must always looks like ~30 (initial) ~30 (first repeat) 55 (second repeat) ~30 55 ~30 55 ~30 55 ~30 55
+  so after second cast timers are always flipped
+ * SPELL_BOMBARD_SLIME should be used out of combat (waypoint script?)
+ * He should call all stitched giants after aggro
+ */
+
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
+#include "PassiveAI.h"
+#include "ObjectAccessor.h"
 #include "SpellScript.h"
 #include "SpellAuraEffects.h"
 #include "naxxramas.h"
```

Comment:
while you are at it, you could also reorder these header includes as intended, that is shared header on top an the rest on alphabetical. This means ``naxxramas.h` should be on top.